src/components/KExternalLink/KExternalLink.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <a
+    v-if="isHrefValid"
+    class="k-external-link"
+    :href="href"
+    rel="noopener"
+    target="_blank"
+    @mouseleave="isMouseOver = false"
+    @mouseover="isMouseOver = true"
+  >
+    <slot />
+    <KIcon
+      v-if="!hideIcon"
+      :color="iconColor"
+      icon="externalLink"
+      size="12"
+    />
+  </a>
+</template>
+
+<script lang="ts">
+const isValidUrl = (url: string): boolean => {
+  try {
+    // eslint-disable-next-line no-new
+    new URL(url)
```

Comment:
You shouldn't need the eslint comment if you set it to a variable

Reply:
Then I get the unused variable error lol. Either way need eslint comment